### DISCUSSION TITLE
IFCObject Placement of a new element with arbitary shape generated from a point cloud

### DISCUSSION THREAD

**Discussion by mouasda** [IfcOpenShell/IfcOpenShell#4131] [General]
Hi, i am trying to add an IFC element with an arbitrary shape (Representation: Tessalation) to an existing IFC file. Is there a generic method to add an element with an arbitary shape to IFC coordinate system?

**Comment by Moult** [comment]
https://blenderbim.org/docs-python/autoapi/ifcopenshell/api/geometry/add_mesh_representation/index.html
